src/metabase/notification/send.clj

```diff
@@ -62,12 +63,17 @@
                               (try
                                 (channel/send! channel message)
                                 (catch Exception e
-                                  (when (should-retry-sending? e (:type channel))
+                                  (let [skip-retry? (should-skip-retry? e (:type channel))
+                                        new-e       (ex-info (ex-message e)
```

Comment:
Why not pass `e` as a third argument to retain the full stack?

Reply:
done!